projects/kit/components/toggle/toggle-options.ts

```diff
@@ -0,0 +1,28 @@
+import {InjectionToken} from '@angular/core';
+import {TuiContextWithImplicit} from '@taiga-ui/cdk';
+import {TuiSizeL} from '@taiga-ui/core';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export interface ToggleOptions {
+    readonly icons: Readonly<{
+        toggleOff: PolymorpheusContent<TuiContextWithImplicit<TuiSizeL>>;
+        toggleOn: PolymorpheusContent<TuiContextWithImplicit<TuiSizeL>>;
+    }>;
+}
+
+/** Default values for the toggle options. */
+export const TUI_TOGGLE_DEFAULT_OPTIONS: ToggleOptions = {
+    icons: {
+        toggleOff: ({$implicit}) =>
```

Comment:
Please change it to actual function instead of an arrow (see [here](https://github.com/TinkoffCreditSystems/taiga-ui/blob/main/projects/core/components/primitive-checkbox/checkbox-options.ts#L29)). Otherwise Angular creates a separate private export with auto-generated name for it and we get naming conflict because of many secondary entry points.

Reply:
Interesting behavior. I've changed it